src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/ReferenceList.cs

```diff
@@ -24,38 +24,42 @@ public int Count
             get { return _references.Count; }
         }
 
-        public int Add(object value!!)
+#pragma warning disable CS8995 // Nullable type 'object?' is null-checked and will throw if null.
+        public int Add(object? value!!)
```

Comment:
Ah, yes, this is an interesting one!  `Add` implements `Add` from `IList`. `IList` says `Add(object? value)`.
So, this signature must match `IList`, but when it does, then a `CS8995` compilation error occurs!

So, either way, there's a compilation error:
* without the
`error CS8767: Nullability of reference types in type of parameter 'value' of 'int ReferenceList.Add(object value)' doesn't match implicitly implemented member 'int IList.Add(object? value)' (possibly because of nullability attributes).`

* without the supression:
`error CS8995: Nullable type 'object?' is null-checked and will throw if null`

Reply:
`IList` again..